src/renderer/components/MainPage.tsx

```diff
@@ -438,6 +438,22 @@ class MainPage extends React.PureComponent<Props, State> {
 
         const activeServer = this.state.servers.find((srv) => srv.id === this.state.activeServerId);
 
+        let titleBarSpacing;
+        if (window.process.platform !== 'darwin') {
+            if (this.state.fullScreen && window.process.platform !== 'linux') {
+                titleBarSpacing = (
+                    <div
+                        className={`button full-screen-button${this.props.darkMode ? ' darkMode' : ''}`}
+                        onClick={this.handleExitFullScreen}
+                    >
+                        <i className='icon icon-arrow-collapse'/>
+                    </div>
+                );
+            } else {
+                titleBarSpacing = (<span style={{width: `${window.innerWidth - (window.navigator.windowControlsOverlay?.getTitlebarAreaRect().width ?? 0)}px`}}/>);
```

Comment:
So, does this mean that Linux doesn't need this special case for full screen?

Reply:
At this point, no. I spent some time trying to work around it but full screen under Linux refuses to behave nicely. So i'm going to create a separate PR that removes full screen support from Linux.